packages/@webex/plugin-cc/src/WebSocket/types.ts

```diff
@@ -0,0 +1,52 @@
+import {WebSocketEvent} from '../types';
+
+// ts doc
+/**
+ * Interface for WebSocket
+ */
+interface IWebSocket {
+  /**
+   * Subscribe to the WebSocket events
+   * @param {string} event
+   * @param {function} callback
+   * @returns {void}
+   */
+  on(event: string, callback: (event: WebSocketEvent) => void): void;
+  /**
+   * Unsubscribe from the WebSocket events
+   * @param {string} event
+   * @param {function} callback
+   * @returns {void}
+   */
+  off(event: string, callback: (event: WebSocketEvent) => void): void;
+  /**
+   * Subscribe and connect to the WebSocket
+   * @param {object} params
+   * @param {string} params.datachannelUrl
+   * @param {object} params.body
+   * @returns {Promise<void>}
+   */
+  subscribeAndConnect(params: {datachannelUrl: string; body: object}): Promise<void>;
```

Comment:
Shouldn't the body here be SubscribeRequest

Reply:
addressed